mmdet/datasets/pipelines/loading.py

```diff
@@ -384,6 +385,100 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class LoadPanopticAnnotations(LoadAnnotations):
```

Comment:
May need to update `tools/misc/browse_dataset.py` to support visualization

Reply:
I will do this after supporting the visualization for panoptic dataset.